app/views/users/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h2>Register a New User</h2>
+
+<%= form_with url: register_path, method: :post, local: true do |form| %>
+  <%= form.label :user_name %>
+  <%= form.text_field :user_name %><br>
+  <%= form.label :email %>
+  <%= form.text_field :email %><br>
+  <%= form.submit "Create New User" %>
+<% end %>
```

Comment:
just a thought to refactor in the future with a model binded(?) bound(?) to this form!